<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PokerTable</class>
 <widget class="QMainWindow" name="PokerTable">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>852</width>
    <height>609</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>PokerTable</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="raise">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>470</y>
      <width>131</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Raise</string>
    </property>
   </widget>
   <widget class="QPushButton" name="fold">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>530</y>
      <width>131</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Fold</string>
    </property>
   </widget>
   <widget class="QPushButton" name="checkButton">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>500</y>
      <width>131</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Call</string>
    </property>
   </widget>
   <widget class="QPushButton" name="nextPlayer">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>440</y>
      <width>51</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Next</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="textBrowser">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>450</y>
      <width>341</width>
      <height>111</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="comLabel2">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>160</y>
      <width>50</width>
      <height>80</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="comLabel4">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>160</y>
      <width>50</width>
      <height>80</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="comLabel1">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>160</y>
      <width>50</width>
      <height>80</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="comLabel3">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>160</y>
      <width>50</width>
      <height>80</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="comLabel5">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>160</y>
      <width>50</width>
      <height>80</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="playerLab1_1">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>320</y>
      <width>50</width>
      <height>80</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="playerLab1_2">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>320</y>
      <width>50</width>
      <height>80</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="playerLab2_2">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>350</y>
      <width>50</width>
      <height>80</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="playerLab2_1">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>350</y>
      <width>50</width>
      <height>80</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="playerLab3_1">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>240</y>
      <width>50</width>
      <height>80</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="playerLab3_2">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>240</y>
      <width>50</width>
      <height>80</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="playerLab6_1">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>40</y>
      <width>50</width>
      <height>80</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="playerLab6_2">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>40</y>
      <width>50</width>
      <height>80</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="playerLab7_1">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>10</y>
      <width>50</width>
      <height>80</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="playerLab7_2">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>10</y>
      <width>50</width>
      <height>80</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="playerLab10_1">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>350</y>
      <width>50</width>
      <height>80</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="playerLab9_2">
    <property name="geometry">
     <rect>
      <x>770</x>
      <y>240</y>
      <width>50</width>
      <height>80</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="playerLab10_2">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>350</y>
      <width>50</width>
      <height>80</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="playerLab8_2">
    <property name="geometry">
     <rect>
      <x>770</x>
      <y>120</y>
      <width>50</width>
      <height>80</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="playerLab8_1">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>120</y>
      <width>50</width>
      <height>80</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="playerLab9_1">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>240</y>
      <width>50</width>
      <height>80</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="playerLab5_1">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>10</y>
      <width>50</width>
      <height>80</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="playerLab5_2">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>10</y>
      <width>50</width>
      <height>80</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="playerLab4_2">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>120</y>
      <width>50</width>
      <height>80</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="playerLab4_1">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>120</y>
      <width>50</width>
      <height>80</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBox">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>440</y>
      <width>71</width>
      <height>26</height>
     </rect>
    </property>
    <property name="maximum">
     <number>1000000</number>
    </property>
    <property name="singleStep">
     <number>5</number>
    </property>
    <property name="value">
     <number>5</number>
    </property>
   </widget>
   <widget class="QPushButton" name="startButton">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>450</y>
      <width>191</width>
      <height>101</height>
     </rect>
    </property>
    <property name="text">
     <string>Start game</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>852</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
